lib/rails_translation_manager/locale_checker/actual_available_locales.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ActualAvailableLocales < BaseChecker
+    include LocaleCheckerHelper
+  
+    def report
+        diff = compare
+        "- \e[31m[ERROR]\e[0m Please check the following locale files exist: '#{diff}' <link to future documentation>" unless diff.empty?
```

Comment:
We already know the locale files don't exist - I'd reword the error message to something more useful. I'd also split out each missing locale, like we've done for the [other checkers](https://github.com/alphagov/rails_translation_manager/blob/52d262b2b6e246e3570afbb18218d5840736d3bf/lib/rails_translation_manager/locale_checker/incompatible_plurals.rb#L12-L32).

I'd suggest the output should look something like:

```
No locale files detected for:

- pt

Either create these locale files or remove these locales from your I18n `available_locales` config.
```

Reply:
Yeah I agree the message could be improved, I'll change to something like your suggestion 👍